src/lib.js

```diff
@@ -106,6 +106,10 @@ exports.isArray = Array.isArray || function(obj) {
     return ObjProto.toString.call(obj) === '[object Array]';
 };
 
+exports.isSet = function(obj) {
+    return ObjProto.toString.call(obj) === '[object Set]';
```

Comment:
What about `obj && obj[Symbol.toStringTag] == 'Set'`? Does not require shims to patch `Object#toString` (I would argue that they should, and most of them do), but requires `Symbol` polyfill.